App/reducers/reducers.js

```diff
@@ -1,69 +1,90 @@
 import { combineReducers } from 'redux';
 
 const movies = (state = [], action) => {
-
   switch (action.type) {
-    // case 'GET_MOVIES_REQUEST':
-    // case 'CREATE_MOVIE_REQUEST':
-    //   return Object.assign({}, state, {loading: true})
-    //   break;
-    case 'GET_MOVIES_SUCCESS':
-      // return Object.assign({}, state, {movies: action.response})
-      return [...action.response];
-    // case 'GET_MOVIES_FAILURE':
-    // case 'CREATE_MOVIE_SUCCESS':
-    //   return Object.assign({}, state, {loading: true})
-    //   break;
-    // case 'CREATE_MOVIE_FAILURE':
-    //   return Object.assign({}, state)
-    //   break;
-    // case 'DELETE_ALL_REQUEST':
-    // case 'DELETE_ALL_SUCCESS':
-    // return Object.assign({}, state, {loading: true})
-    // break;
-    case 'LOGOUT':
-      return [];
-    default:
+    // case 'ADD_MOVIES':
+    //   return ({ ...state, ...action.movies });
+    // case 'SEEN_MOVIE':
+    //   const newState = { ...state }
+    //   newState[action.id] = Object.assign({}, newState[action.id], {
+    //     seen: true
+    //   });
+    //   return newState;
 
-  }
+    // case 'GET_MOVIES_LOADING':
+    //   if (action.isLoading) return ({ ...state, ...{isLoading: true}});
+    //   return state;
 
-  return state;
-}
+    case 'GET_MOVIES_RECEIVED':
+      if (action.data) {
+        let parsedMovies = parseMovies(action.data);
+        // console.log('parsedMovies', parsedMovies);
+        return parsedMovies;
+      }
+      return state;
 
-const user = (state = {}, action) => {
-  switch (action.type) {
-    case 'LOGIN_SUCCESS':
-      return {...action.response};
-    case 'LOGOUT':
-      return {};
-    default:
-  }
-  return state;
-}
+    case 'GET_MOVIES_ERROR':
+      console.log('ERROR IN REDUCERS:', action.error);
+      return state;
 
-const songs = (state = [], action) => {
-  switch (action.type) {
-    case 'GET_SONGS_SUCCESS':
-      return [...action.response];
-    case 'LOGOUT':
-      return [];
     default:
+      return state;
   }
-  return state;
 }
 
-const flag = (state = true, action) => {
-  switch (action.type) {
-    case 'CHANGE_FLAG':
-      console.log('in the reducer flag');
-      return !state;
-    default:
-  }
-  return state;
+// const user = (state = {}, action) => {
```

Comment:
Same as above. Keep it clean, "saving it for later" is not a reason to keep this in here.